examples/server/server.c

```diff
@@ -1729,7 +1729,10 @@ THREAD_RETURN WOLFSSL_THREAD server_test(void* args)
             != WOLFSSL_SUCCESS)
         err_sys_ex(catastrophic, "unable to load static memory and create ctx");
 #else
-    ctx = SSL_CTX_new(method(NULL));
+    else {
```

Comment:
Instead of the `else` can we just do a check for `if (method)` here instead? With the static memory block above its confusing and prone to possible issues in the future.